ros_buildfarm/templates/snippet/publisher_warnings.xml.em

```diff
@@ -1,49 +1,44 @@
-    <hudson.plugins.warnings.WarningsPublisher plugin="warnings@@4.68">
-      <healthy/>
-      <unHealthy/>
-      <thresholdLimit>low</thresholdLimit>
-      <pluginName>[WARNINGS] </pluginName>
-      <defaultEncoding/>
-      <canRunOnFailed>false</canRunOnFailed>
-      <usePreviousBuildAsReference>false</usePreviousBuildAsReference>
-      <useStableBuildAsReference>false</useStableBuildAsReference>
-      <useDeltaValues>false</useDeltaValues>
-      <thresholds plugin="analysis-core@@1.96">
+<io.jenkins.plugins.analysis.core.steps.IssuesRecorder plugin="warnings-ng@@7.3.0">
+  <analysisTools>
+    <!--
+         catkin inject CMake variables that are not
+         always used in build. Be careful if planning
+         to add the CMake parser
+    -->
```

Comment:
The CMake parser was used before. Why not anymore?

Reply:
Uh. When I saw the warning coming from `Manually-specified variables were not used by the project: BUILD_TESTING` I thought there was no CMake warning parser before. I was wrong it was enabled (thanks for catching it). 

I've backed to enable the CMake parser in f5c9293 and added an exception in cfb8fbc . The plugin detect the message `Manually-specified variables were not used by the project:` but not the specific variable so I think we can not be more precise on the exclusion. Since the buildfarm code control the CMake variables passed to the cmake invocation, I would say that is ok to go with the "manually-specified variables" exception.